ci/tox/Dockerfile

```
@@ -5,7 +5,7 @@ RUN ln -s /usr/local/bin/python /usr/bin/python
 
 ARG git_version
 RUN set -x \
-    && apk add --no-cache --virtual git-build-deps  alpine-sdk autoconf gettext wget zlib-dev \
+    && apk add --no-cache --virtual git-build-deps alpine-sdk autoconf gettext wget zlib-dev \
```

Maybe:
```suggestion
    && apk add --no-cache --virtual=git-build-deps alpine-sdk autoconf gettext wget zlib-dev \
```
if it works